plugins/BEdita/Core/src/Model/Entity/Stream.php

```diff
@@ -248,4 +254,16 @@ protected function _getUrl()
 
         return $this->_properties['url'] = FilesystemRegistry::getPublicUrl($this->uri);
     }
+
+    protected function _getMetadata($source)
+    {
+        $ext_allowed = Configure::read('Streams.ext_allowed');
```

Comment:
I'd keep a list of allowed mime types in a class constant for now, like `EXIF_MIME_TYPES`